xcodeproj/files.py

```diff
@@ -18,7 +19,8 @@ class PBXBuildFile(PBXObject):
     effectively pointers to a PBXFileReference which is an actual file.
     """
 
-    file_ref_id: str
+    file_ref_id: Optional[str]
```

Comment:
Note: This will be a breaking change so we will have to bump the major version on release.

Reply:
Makes sense, CC @guojiubo

Reply:
Let's bump to 1.0.0 then.